riscv/src/lib.rs

```diff
@@ -8,34 +8,140 @@ use std::{
     process::Command,
 };
 
-use powdr_number::FieldElement;
+use powdr_number::KnownField;
 use std::fs;
 
-pub use crate::runtime::Runtime;
-
 mod code_gen;
+mod code_gen64;
 pub mod continuations;
 pub mod elf;
 pub mod runtime;
+pub mod runtime64;
 
 static TARGET_STD: &str = "riscv32im-risc0-zkvm-elf";
 static TARGET_NO_STD: &str = "riscv32imac-unknown-none-elf";
 
+#[derive(Default, Clone)]
+pub struct RuntimeLibs {
+    pub arith: bool,
+    pub keccak: bool,
+    pub poseidon: bool,
+}
+
+impl RuntimeLibs {
+    pub fn new() -> Self {
+        Self {
+            arith: false,
+            keccak: false,
+            poseidon: false,
+        }
+    }
+
+    pub fn with_arith(self) -> Self {
+        Self {
```

Comment:
you can do the following:
```
Self {
    arith: true,
    ..self
}
```
and it will move all the other fields to the new object